ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/EventsRange.scala

```diff
@@ -52,6 +57,36 @@ object EventsRange {
     else EmptyEventSeqIdRange.copy(endInclusive = readEventSeqId(endInclusive)(connection))
   }
 
+  private def readUpperBound(endInclusive: Offset)(connection: java.sql.Connection): Long =
```

Comment:
this method is gone in the master and I believe unused in this branch. `readEventSeqId` is used for reading both upper and lower bounds

Reply:
Oops, bad merge conflict resolution I guess. Thanks.

Reply:
cea2837993